### PR TITLE
New Tutorial Notebook for Hackathon and model changes

### PR DESCRIPTION
**Pull request by zparis99**
There are many changes here:

1. A new notebook for the hackathon with a pretty in-depth intro with a lot of guided steps of what can be worked on. Notebook also stands on its own more now.
2. Adds a batch norm layer with no learned affine transformations (so just for normalization) for the model inputs to make sure that the input data is actually normalized.
3. Changes loss function to a weighted sum of - pearson correlation and mse loss to try and get benefits of each.
4. Various other bug fixes and cleaning up of old code.

I'm still confirming that all of this will work as is for a new user, but wanted to get it ready for you to look at the notebook to let me know if you think it's at a good place for the hackathon. Let me know! I think there are likely still other bug-fixes to be made but I will work on those more tomorrow.

One other thing we should do before the hackathon is replace the main branch on the main repo with the mae-st branch (this one). This branch is much more up-to-date and likely will work better going forward so should be the default.

### CODE REVIEW

File: notebooks/ECoG_Model_Training.ipynb
Comment (itamarJa) [inline notebooks/ECoG_Model_Training.ipynb:1]: I'll run this early next week. Looks great, but we should probably separate it into multiple notebooks so that it's more manageable. I'll let you know once I start running it.

**Review by itamarJa — APPROVED** [review]

File: notebooks/ECoG_Model_Training.ipynb
Comment (zparis99) [reply notebooks/ECoG_Model_Training.ipynb:1]: Sounds good! I'm going to push a few more updates that make it work better with a gpu, this one still will not work right with a colab notebook, I'll push that tomorrow.
